### PR TITLE
fix(#7552): Fix notebook snapshot image annotations

### LINKED ISSUE #7552: Annotating image notebook snapshots is broken
#### Summary

Opening an image snapshot and annotating it causes Open MCT to crash.

#### Expected vs Current Behavior

Should be able to annotate image snapshots without failure

#### Steps to Reproduce

1. Create Example Imagery
2. Open an Image
3. Select Snapshot > Create a Notebook Snapshot
4. Open the Snapshots menu from the Snapshot indicator
5. Click the Image
6. In the context menu, click "annotate"
7. No image, errors in console

#### Environment

* Open MCT Version: 
* Deployment Type: 
* OS:
* Browser:

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by ozyx**
Closes #7552 

### Describe your changes:

Fixes imports for `painterro` as it is exporting CJS, and this is now ESM.

Needed to change:

```js
this.painterro = Painterro(this.config);
```
to
```js
this.painterro = Painterro.default(this.config);
```

- Adds minimal e2e smoke test for notebook snapshot image annotation

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js [hunk 1]
```diff
@@ -90,23 +90,35 @@ test.describe('Snapshot Container tests', () => {
...
+  }) => {
+    test.info().annotations.push({
+      type: 'issue',
+      description: 'https://github.com/nasa/openmct/issues/7552'
+    });
+    await page.getByLabel('My Items Notebook Embed').getByLabel('More actions').click();
+    await page.getByRole('menuitem', { name: 'View Snapshot' }).click();
+    await expect(page.getByRole('dialog', { name: 'Modal Overlay' })).toBeVisible();
+    await page.getByTitle('Annotate').click();
+    await expect(page.locator('#snap-annotation-canvas')).toBeVisible();
```
Comment (unlikelyzero) [inline e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js:107]: can we do aria-describedby the dom id, here?

Comment (ozyx) [reply e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js:107]: I think that's coming from painterro itself? I don't see it in our code. still looking

File: e2e/README.md [hunk 2]
```diff
@@ -516,6 +516,30 @@ test.describe('foo test suite', () => {
 - Working with multiple pages
 There are instances where multiple browser pages will needed to verify multi-page or multi-tab application behavior. Make sure to use the `@2p` annotation as well as name each page appropriately: i.e. `page1` and `page2` or `tab1` and `tab2` depending on the intended use case. Generally pages should be used unless testing `sharedWorker` code, specifically.
 
+- Working with file downloads and JSON data
+Open MCT has the capability of exporting certain objects in the form of a JSON file handled by the chrome browser. The best example of this type of test can be found in the exportAsJson test.
```
Comment (ozyx) [inline e2e/README.md:520]: This is great, although I'm worried that without something specific to our e2e testing framework we're just reproducing the Playwright docs here?

File: e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js [hunk 3]
```diff
@@ -72,41 +72,90 @@ test.describe('Snapshot Container tests', () => {
...
+      page.waitForEvent('download'), // Waits for the download event
+      page.getByLabel('Export as JPG').click() // Triggers the download
+    ]);
+
+    // Save as PNG
+    await expect(page.getByLabel('Modal Overlay').getByRole('img')).toBeVisible();
+    await Promise.all([
+      page.waitForEvent('download'), // Waits for the download event
+      page.getByLabel('Export as PNG').click() // Triggers the download
+    ]);
```
Comment (ozyx) [inline e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js:157]: awesome!

File: src/plugins/timelist/TimelistComponent.vue [hunk 4]
```diff
@@ -38,39 +38,41 @@
...
-              @sort="sort"
+    <template v-else>
+      <div class="c-table c-table--sortable c-list-view c-list-view--sticky-header sticky">
+        <table class="c-table__body js-table__body">
+          <thead class="c-table__header">
+            <tr>
+              <list-header
+                v-for="headerItem in headerItems"
+                :key="headerItem.property"
+                :direction="
```
Comment (jvigliotta) [inline src/plugins/timelist/TimelistComponent.vue:49]: I know this was in here from before, but can we move this logic out of the template? Maybe we make headerItems a computed, something like:

```js
headerItems() {
    return headerItems.map(item => ({
      ...item,
      direction: this.defaultSort.property === item.property
        ? this.defaultSort.defaultDirection
        : item.defaultDirection
    }));
  }

**Review by jvigliotta — CHANGES_REQUESTED** [review]
Looks great, one suggestion.

File: src/plugins/timelist/TimelistComponent.vue (hunk 4 above)
Comment (ozyx) [reply src/plugins/timelist/TimelistComponent.vue:49]: I like it!

**Review by unlikelyzero — APPROVED** [review]
LGTM

**Review by jvigliotta — APPROVED** [review]
Looks great! Nice work.

**Pull request merged**